contents/docs/privacy/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Privacy compliance
+sidebarTitle: Overview
+sidebar: Docs
+showTitle: true
+---
+
+PostHog offers considerable flexibility in hosting and configuration options to comply with privacy regulations around the world.
+
+In these guides, we offer advice for using PostHog in a compliant manner under the following legal frameworks:
+
+- [The General Data Protection Regulation](/docs/privacy/gdpr-compliance/) (GDPR), which applies to all businesses collecting data on EU citizens
+
+- [The Health Insurance Portability and Accountability Act](/docs/privacy/hipaa-compliance/) (HIPAA), which applies to businesses capturing and processing health data in the US
+
+- [The California Consumer Privacy Act](/docs/privacy/ccpa-compliance/) (CCPA), which applies to qualifying for-profit businesses collecting personal information on residents of California
+
+**Please note:** these guides don't constitute legal advice. We recommend seeking professional advice to ensure you remain compliant with relevant legislation.
+
+In addition to these guides, we also provide ongoing [updates and advice on privacy issues](/blog/categories/privacy) on the PostHog blog.
+
+### Further reading
```

Comment:
I'd take this section out - we should keep our docs purely focused on our product and not sending people outside of them to SEO pages (linking the other way is good, but I wouldn't go Docs -> Blog)